lib/amorail/entity.rb

```diff
@@ -110,15 +110,14 @@ def commit_request(attrs)
 
     def handle_response(response, method)
       return false unless response.status == 200
```

Comment:
It looks like we can receive 204 only on GET request when we try to find object(s) in amoCRM.
For instance, this method would be called with 204 status when we call `Entity#find` with nonexistent id.
In this case we would return `false` and then return `nil` from `#load_record`. 

I think (but I'm not sure) we can't receive 204 on POST request because it doesn't make sense.
Looks like we can left it as it is now. 
What do you think?